synapse/handlers/register.py

```diff
@@ -59,7 +57,6 @@ def __init__(self, hs):
         self._auth_handler = hs.get_auth_handler()
         self.profile_handler = hs.get_profile_handler()
         self.user_directory_handler = hs.get_user_directory_handler()
-        self.captcha_client = CaptchaServerHttpClient(hs)
```

Comment:
CaptchaServerHttpClient is dead too now.

you gotta chase it all the way... :)

(I think that's the end of it tbh)